js/clipboardToggleAll.js

```diff
@@ -0,0 +1,39 @@
+(function ($)
+  {
+    // Toggle buttons
+    var clickClipButton = function (node, selectAll) {
+      $button = $(node);
```

Comment:
TIP: always use `var` to make sure that you are declaring a new variable within the scope of that function and not assigning a new value to an existing variable defined outside of that scope ([more here](https://github.com/getify/You-Dont-Know-JS/tree/master/scope%20%26%20closures)).